openwisp_controller/config/views.py

```diff
@@ -23,7 +23,12 @@ def get_default_templates(request, organization_id):
     backend = request.GET.get("backend", None)
     user = request.user
     authenticated = user.is_authenticated
-    if not authenticated and not user.is_staff:
+    if (
+        not authenticated
+        or not user.is_staff
+        or (not user.is_superuser and user.is_manager(organization_id))
```

Comment:
shouldn't it be as follows?

```python
or (not user.is_superuser and not user.is_manager(organization_id))
```

I did some manual testing with a staff user, non superuser, which has access only to one org, but I was still able to access the URL for the other orgs in the system.

Reply:
Yes you are right. This was an overlook.

>  I did some manual testing with a staff user, non superuser, which has access only to one org, but I was still able to access the URL for the other orgs in the system.

Did you also test after making that suggested change? After making this change, it worked fine as expected.